### DISCUSSION TITLE
2024/01/14/vim-marcos/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#10] [General]
# 2024/01/14/vim-marcos/

“简单来说，normal模式中按下字母q进入录制模式，此时的操作会被记录。再次按下q录制完成。要执行已录制好的宏，只需要键入n@a即可，其中n是执行次数，a是宏名称”

https://wz.anoms.top/2024/01/14/vim-marcos/

**Comment by Hazelandinstant** [comment]
哇！这个牛！但是有点没理解，是不是意味着如果你经常重复性操作某个软件，总是使用该软件的某个功能，于是就可以通过宏来 程序化内容，下次只要重复宏操作，就可以实现 使用 某个软件的某个功能了？
在mac系统中有一个软件好像作用相似，Keyboard Maestro，我自己初初尝试过，或许没准是宏的GUI版本

**Reply by cyclinbox** [reply]
确实是的！宏相当于把软件的重复操作给存起来，以后就可以自动进行了。不过宏能够记录的只有软件内的操作。
我去看了一下Keyboard Maestro，好像它是那种系统级别的自动化操作工具，记录的操作可以跨软件，这个有点厉害了，可能算是宏++版本（？）
